drivers/modem/Kconfig.gsm

```diff
@@ -61,4 +61,13 @@ config MODEM_GSM_MANUAL_MCCMNO
 	  the network provider and may need to be changed if auto is not
 	  selected.
 
+config MODEM_GSM_FACTORY_RESET_AT_BOOT
+	bool "Factory reset modem at boot"
+	default n
```

Comment:
No need to disable the option as they are disabled by default

Reply:
done